vendor/bat-native-ads/include/bat/ads/ads.h

```diff
@@ -36,6 +36,9 @@ using GetStatementCallback =
 // development servers but can be overridden via command-line arguments
 extern Environment _environment;
 
+// |_sys_info| contains the hardware |manufacturer| and |model|
+extern SysInfo _sys_info;

```

Comment:
see https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

Reply:
The convention for globals is `g_name`

Reply:
Agreed will fix the g_ for globals, just kept consistent from legacy.